src/lib/analytics/analytics.js

```diff
@@ -185,7 +185,17 @@ const patchLogger = () => {
       global.Rollbar.error(logMessage, errorObj, { logContext, eMsg, rest })
     }
     if (Config.sentryDSN && Config.env !== 'test') {
-      reportToSentry(errorObj && errorObj instanceof Error ? errorObj : new Error(logMessage), {
+      const isValidErrorObject = errorObj && errorObj instanceof Error
```

Comment:
1. just instanceof check will be enough 
2. why about UnhandledRejection ? Why & where it appears ? According to the ` captured with keys: #, @, _, err, out` i could suppose it's something related to the Gun

Reply:
@serdiukov-o-nordwhale these changes is not related to the main problem of #1755 issue. This is the improvement for sentry log message and extra data.
The error message captured with keys: #, @, _, err, out - is a 100% gun issue. But I don't know why it is unhandled in this specific case.